index.html

```diff
@@ -18,14 +19,16 @@
         </div>
         <div class="form-field">
           <label for="level" class="form-label">Level*</label>
-          <select name="level" class="form-input">
+          <select id="level" name="level" class="form-input">
             <option value="basic">Basic</option>
             <option value="advanced">Advanced</option>
           </select>
         </div>
         <div class="form-field">
           <label for="hoursPerWeek" class="form-label">Hours per week*</label>
+          :)
           <input
+            id="hoursperweek"
```

Comment:
Remember to use camelCase correctly - a capital letter is needed for each subsequent word after the first: `hoursPerWeek`. Please update this to meet JavaScript conventions.